PostHog/posthog
# feat(onboarding-1.0): Ingestion wizard panel footers and headers

## Problem



## Changes

updates to new ingestion wizard headers and footers

<img width="1185" alt="Screen Shot 2022-04-22 at 5 28 19 PM" src="https://user-images.githubusercontent.com/25164963/164798147-a241e6e6-d223-4a09-a368-44c0f2489d6a.png">


https://user-images.githubusercontent.com/25164963/164798216-ec0a65b0-1ba8-4b47-8cdb-78cb16adb8d1.mov






👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

- set up feature flag `onboarding-1`
- try the flow with and without feature flag on
- just see if nothing is broken
